src/nuget/project.json

```diff
@@ -0,0 +1,27 @@
+{
+  "version": "1.0.0-featmsbuild-*",
+  "buildOptions": {
+    "emitEntryPoint": true
+  },
+  "dependencies": {
+    "Microsoft.NETCore.App": {
+      "type": "platform",
+      "version": "1.0.0"
+    },
+    "NuGet.Commands": {
```

Comment:
NuGet.CommandLine.XPlat references this.  Why do we need an explicit reference to it?